tgui/packages/tgui/interfaces/common/LoadingScreen.tsx

```diff
@@ -0,0 +1,30 @@
+import { Icon, Stack } from 'tgui-core/components';
```

Comment:
this file is just renamed really. i removed the dimmer, so it doesnt flash so much when the screen is loading